src/main/java/flaxbeard/steamcraft/handler/SteamcraftTickHandler.java

```diff
@@ -33,6 +35,86 @@
     private float sensitivity = 0;
     private int zoomSettingOn = 0;
     private boolean lastPressingKey = false;
+    private boolean isJumping = false;
+
+    @SubscribeEvent
+    public void onPlayerTick(TickEvent.PlayerTickEvent event) {
+        EntityPlayer player = event.player;
+        World world = player.worldObj;
+        boolean isClient = event.side == Side.CLIENT;
+        boolean isServer = event.side == Side.SERVER;
```

Comment:
Don't you really just need one boolean here? :c I mean, it's either client or server, it's never both as far as I know.